fbchat/models.py

```diff
@@ -526,6 +591,12 @@ def __repr__(self):
         # For documentation:
         return '{}.{}'.format(type(self).__name__, self.name)
 
+class QuickReplyType(Enum):
```

Comment:
There's not really a need for this model any longer

Reply:
It's needed because of
```python
q["content_type"] = quick_reply.type.value.lower()
```
in `_getSendData`